### PR TITLE
Changes in volume_adjust

### PR DESCRIPTION
**Pull request by EstrellaFG**
Changes in volume_adjust and subtraction protocols in order to meet the suggestion in the review of the paper. 
Basically include option for compute or not radial average and energy.

### CODE REVIEW

File: xmipp3/protocols/protocol_volume_adjust_sub.py [hunk 1]
```diff
@@ -218,6 +224,8 @@ def _summary(self):
             summary.append("Input mask 2: %s" % self.mask2.get().getFileName())
         if self.resol.get() != 0:
             summary.append("Subtraction at resolution %f A" % self.resol.get())
+        if self.radavg.get() is True:
```
Comment (DStrelak) [inline xmipp3/protocols/protocol_volume_adjust_sub.py:227]: `if condition is True:` == `if condition`.
Also, why do you have `self.radavg.get()` here and `self.radavg` on line 205 and 294?

File: xmipp3/protocols/protocol_volume_adjust_sub.py [hunk 2]
```diff
@@ -295,6 +305,8 @@ def _summary(self):
             summary.append("Input mask 2: %s" % self.mask2.get().getFileName())
         if self.resol.get() != 0:
             summary.append("Filter at resolution %f A" % self.resol.get())
+        if self.radavg.get() is True:
```
Comment (DStrelak) [inline xmipp3/protocols/protocol_volume_adjust_sub.py:308]: same as above

File: xmipp3/protocols/protocol_volume_adjust_sub.py [hunk 3]
```diff
@@ -60,6 +63,10 @@ def _defineParams(self, form):
                       expertLevel=LEVEL_ADVANCED,
                       help='Relaxation factor for Fourier amplitude projector (POCS), it should be between 0 and 1, '
                            'being 1 no relaxation and 0 no modification of volume 2 amplitudes')
+        form.addParam('radavg', BooleanParam, label="Match the rotationally averaged Fourier amplitudes?", default=True,
+                      help='Match the rotationally averaged Fourier amplitudes when adjusting the amplitudes instead of'
+                           ' taking directly them from the reference volume. For subtraction and consensus is '
```
Comment (DStrelak) [inline xmipp3/protocols/protocol_volume_adjust_sub.py:68]: `taking them directly`, `consensus it is recommended`

File: xmipp3/protocols/protocol_volume_adjust_sub.py [hunk 4]
```diff
@@ -60,6 +63,10 @@ def _defineParams(self, form):
...
+                           'recommended to set it to True but for sharpening is recommended to set it to False')
```
Comment (DStrelak) [inline xmipp3/protocols/protocol_volume_adjust_sub.py:69]: `sharpening it is recommended`

**Review by DStrelak — DISMISSED** [review]
I'm not gonna merge it at the moment, as the program is still under review, right?

**Comment by EstrellaFG** [comment]
> I'm not gonna merge it at the moment, as the program is still under review, right?

Yes, and the program is already ready for review

**Review by DStrelak — APPROVED** [review]

**Pull request merged**